source/nodejs/adaptivecards-designer/src/adaptive-card-schema.ts

```diff
@@ -147,6 +147,7 @@ export const adaptiveCardSchema =
           "$ref": "#/definitions/Actions"
         },
         "body": {
```

Comment:
I updated the render method to optionally restore the selected state.

The only issue I ran into was that you need to refresh the layout to update peer location again. I expanded the pointer event listener to watch the entire window instead of just the designer surface, so it is slightly better now - but not the ideal solution.

![Carousel-3](https://user-images.githubusercontent.com/98650930/178789354-a803def1-07fe-4fbf-8ddf-fb157e9d1f04.gif)